contacts24/models/notes.py

```diff
@@ -1,14 +1,91 @@
-def get_notes():
-    pass
 
+from collections import UserDict
 
-def create_note():
-    pass
+class Note(UserDict):
+    """Цей клас має функціонал для роботи з нотатками"""
+    def add_note(self, client_id, title, text):
+        """Додає нову нотатку до записника."""
+        note_id = len(self.data) + 1  # Генерує унікальний ідентифікатор для нотатки
+        self.data[note_id] = {'client_id': client_id, 'title': title, 'text': text}
 
+    def change_note(self, key, new_text):
+        """Змінює текст існуючої нотатки за її ключем."""
+        if key in self.data:
+            self.data[key]['text'] = new_text
+        else:
+            raise KeyError(f"Note with key {key} not found.")
```

Comment:
`KeyError` - @IhorNi  don't we agree to wrap all errors in `AppError`?

Reply:
@SavchukR in baseline I have custom errors in errors.py which produce user-friendly comments for user what exaclty function expects
So that instead of just a wrapped error message it will be smth like
```
class NonExistingContact(InputError):
    def __str__(self):
        return "NonExistingContact: Check contact name, no such contact in address book"